alacritty_terminal/src/term/mod.rs

```diff
@@ -56,6 +56,10 @@ pub trait Search {
     fn semantic_search_left(&self, _: Point<usize>) -> Point<usize>;
     /// Find the nearest semantic boundary _to the point_ of provided point.
     fn semantic_search_right(&self, _: Point<usize>) -> Point<usize>;
+    /// Find the beginning of a potentially wrapped line.
+    fn line_search_left(&self, _: Point<usize>) -> Point<usize>;
```

Comment:
@nixpulvis Any feedback on the function documentation? I feel like it's not 100% clear in what exactly it does.

Reply:
`Find the beginning of a line, following line wraps`?

maybe the function name could be clearer, like `wrap_line_search_left`.

Reply:
I've intentionally removed the `wrap` from the function name, since `wrap_line_search_left` sounds too much like the function actually does something wrapping related. But it actually just follows across linewraps. It's also more consistent with the other search functions for selection, since we're talking about a line selection and not a wrapline selection.

I like the comment suggestion you've made though.

Reply:
`/// Find the beginning of a line or a line that has been wrapped.`, or something similar?

Otherwise, IMHO, the current is fine.

Reply:
That sounds like it's finding a line that has been wrapped. But we're finding the beginning of the line, across potential wrappage. So it's not entirely accurate.

Reply:
I was idling so long on that comment I didn't see @ike's above. I like that wording, or something similar.